src/OpenTK.Mathematics/Vector/Vector2i.cs

```diff
@@ -0,0 +1,603 @@
+/*
+Copyright (c) 2006 - 2008 The Open Toolkit library.
```

Comment:
Give me a ride in your Delorean sometime!

Reply:
(use the standardized header :P)